tooling/cli/node/tauri.js

```diff
@@ -7,7 +7,7 @@
 const cli = require('./main')
 const path = require('path')
 
-const [bin, script, ...arguments] = process.argv
```

Comment:
`arguments` is a reserved keyword in JavaScript and cannot be used as a variable name in strict mode: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments